pkg/kubelet/network/hostport/hostport_syncer_test.go

```diff
@@ -168,16 +168,16 @@ func TestOpenPodHostports(t *testing.T) {
 		false,
 	}
 
-	err := h.OpenPodHostportsAndSync(tests[0].mapping, "br0", activePodPortMapping)
+	err := h.OpenPodHostportsAndSync(tests[0].mapping, activePodPortMapping)
 	if err != nil {
 		t.Fatalf("Failed to OpenPodHostportsAndSync: %v", err)
 	}
 
 	// Generic rules
 	genericRules := []*ruleMatch{
-		{-1, "POSTROUTING", "-m comment --comment \"SNAT for localhost access to hostports\" -o br0 -s 127.0.0.0/8 -j MASQUERADE"},
-		{-1, "PREROUTING", "-m comment --comment \"kube hostport portals\" -m addrtype --dst-type LOCAL -j KUBE-HOSTPORTS"},
-		{-1, "OUTPUT", "-m comment --comment \"kube hostport portals\" -m addrtype --dst-type LOCAL -j KUBE-HOSTPORTS"},
+		{-1, "PREROUTING", "-m comment --comment \"maybe kube hostport\" -m addrtype --dst-type LOCAL -j KUBE-LOCALDEST"},
+		{-1, "OUTPUT", "-m comment --comment \"maybe kube hostport\" -m addrtype --dst-type LOCAL -j KUBE-LOCALDEST"},
+		{-1, "KUBE-LOCALDEST", "-m comment --comment \"maybe kube hostport\" -d 127.0.0.1/32 -j KUBE-HOSTPORTS"},
```

Comment:
This is a little wonky, I admit - the `lo` interface does receive the whole /8, but programmatically I don't want that because, for example, my eth0 reports a /24, but obviously that whole /24 is not local.

So I can either do what I have done, which is to pull the individual /32s of the interfaces, or I can special case localhost as a /8.  Or we can can leave it as it was with dst-type LOCAL and try to find some other way to close this gap overall.  For example, if we ensure that hostports come after services, but that feels brittle to me given that we have 2 different entities managing rules.